README.rst

```diff
@@ -86,8 +86,8 @@ Project goals
 * Build a Pythonic API for GMT.
 * Interface with the GMT C API directly using ctypes (no system calls).
 * Support for rich display in the Jupyter notebook.
-* Integration with the PyData Ecosystem: numpy.ndarray or pandas.DataFrame for data tables
-  and xarray.DataArray for grids.
+* Integration with the `PyData <https://pydata.org/>`__ ecosystem: ``numpy.ndarray`` or ``pandas.DataFrame`` for
+  data tables and ``xarray.DataArray`` for grids.
```

Comment:
Wrap to no more than 80 characters.
```suggestion
* Integration with the `PyData <https://pydata.org/>`__ ecosystem: 
  ``numpy.ndarray`` or ``pandas.DataFrame`` for data tables and 
  ``xarray.DataArray`` for grids.
```

Reply:
Thanks. Did this on GH, so trickier to spot.